src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/osd/osd-performance-histogram/osd-performance-histogram.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, Input, OnChanges, OnInit } from '@angular/core';
+
+import * as _ from 'underscore';
+
+@Component({
+  selector: 'cd-osd-performance-histogram',
+  templateUrl: './osd-performance-histogram.component.html',
+  styleUrls: ['./osd-performance-histogram.component.scss']
+})
+export class OsdPerformanceHistogramComponent implements OnInit, OnChanges {
+  @Input() histogram: any;
+  valuesStyle: any;
+  last = {};
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+  ngOnChanges() {
+    this.render();
+  }
+
+  hexdigits(v): string {
```

Comment:
This looks like rather generic code. Would it make sense to add this to some commonly shared library instead?